app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java

```diff
@@ -57,8 +58,12 @@
 
   private final Stopwatch startupTimer = Stopwatch.createStarted();
 
+  public static void main(String[] args) {
+    LauncherImpl.launchApplication(Shuffleboard.class, ShuffleboardPreloader.class, args);
```

Comment:
How does this affect JUnique?  Should we need to acquire the lock here?

Reply:
Doing the JUnique stuff in `init` lets us show a warning dialog in the future

Reply:
Ok